Ba odd

bernardanderson opened the pull request.

sylviama committed:
integer

Commit:
User Interface Console Porition Nearly Complete
The user interface is nearly complete.  It has been set up to check for a
correct math operation and that the "number of integers" is in fact an
integer.  The tests need to be written.

Commit:
Added Odd class and methods, print code.
The Odd class was added along with the method needed to extract the odd
numbers into List format. The print method was added to the UserInterface
to display returned lists from any of the operation commands.

Commit:
Small updates for the odd class and output.
The odd class removed the stand-alone list return.  Comments were added as
well.

Commit:
Changed WriteLine to Write in number output.

Commit:
Added tests of Odd requirement

Commit:
Added one additional Odd test for the length

sylviama commented:
👍

katerebekah commented on Mathmagician.Tests/OddTests.cs:
```diff
@@ -0,0 +1,70 @@
+
+            //assert 
+            Assert.AreEqual(10000, new Odd().CountOddNumbers(10000).Count);
+        }
```
Add a CollectionAssert.AreEqual test that will compare an expected list to the one your class returns.

katerebekah commented on Mathmagician/UserInterface.cs:
```diff
@@ -0,0 +1,118 @@
+
+namespace Mathmagician
+{
+    public class UserInterface
```
You can unit test the methods in this class, too, but it might not be required.

Commit:
Added test to check returned List<int> from Odd
The test Odd_ReturnedListIsTheCorrectList checks to see if the List<int>
returned from the Odd class is the correct List.

bernardanderson commented:
@katerebekah 
Thanks, I added the test you requested to the ba-odd branch.

katerebekah commented:
:+1:
